src/components/Actions/ProtectedRoute.tsx

```diff
@@ -1,3 +1,5 @@
+// Deprecated
```

Comment:
Delete the file tbh. & probably remove the clutter by taking out all the other deprecated files. It'll be in git history if we ever need to go back to it.